### DISCUSSION TITLE
Advice on using DPGEN for amorphous materials

### DISCUSSION THREAD

**Discussion by ramic-k** [deepmodeling/dpgen#738] [General]
Hi, 

I see that a lot of examples are for the materials with a unit cell, or crystalline materials. I was wondering what would your advice be for example polymers. I want to use DPGEN to create a potential for atactic Polystyrene, and was wondering what would should I aim for when providing the initial structure. Due to use of VASP I assume one cannot go to crazy with the number of atoms? Or if you can point in a direction or an example of DPGEN used for polymer, that would be great as well!

**Comment by AnguseZhang** [comment]
I'm not familiar with polymer simulations. There is one publication which combines coarse graining simulation with DP, https://pubs.rsc.org/en/content/articlelanding/2020/SM/D0SM01019G. There's another work which uses graph neural network to simulate PEG https://pubs.acs.org/doi/abs/10.1021/acs.jpclett.1c02214.  Hope it can help.

**Comment by Revolver233** [comment]
这是来自QQ邮箱的假期自动回复邮件。你好，我最近正在休假中，无法亲自回复你的邮件。我将在假期结束后，尽快给你回复。
